pi5/board.go

```diff
@@ -449,11 +320,7 @@ func (b *pinctrlpi5) DigitalInterruptByName(name string) (board.DigitalInterrupt
 
 // AnalogNames returns the names of all known analog pins.
 func (b *pinctrlpi5) AnalogNames() []string {
-	names := []string{}
-	for k := range b.analogReaders {
-		names = append(names, k)
-	}
-	return names
+	return []string{}
```

Comment:
is there any reason for returning the empty string slice? Wouldn't it just make sense to return an error or something else?

Reply:
yes it might make sense to return something else, but unfortunately this function is a part of the `board` interface (you can look at the whole interface in rdk `components/board/board.go`). All board components must implement these functions so we have to return a string array.

Reply:
Alas, you can't return an error and implement the interface from `board`. Returning the empty list seems like the correct choice to me, to indicate that we do not have analog readers and did not have errors determining that.